cogs/ahk/help.py

```diff
@@ -352,6 +366,14 @@ async def on_open_message(self, message):
 			log.info(f'Opening new channel after claim: {channel}')
 			await self.open_channel(channel)
 
+
+		# update recent ask messages
+		for msg in self.ask_messages:
+			await msg.edit(content=EDITED_ASK_MESSAGE.format(GET_HELP_CHAN_ID))
+		
+		#clear the list since all of them have been resolved now.
```

Comment:
What if another channel is claimed right after? I think a solution where the last three .ask messages are held in the self.ask_messages cache will be the best solution. Simply clearing the list after updating the messages once can lead to the messages being outdated and wrong very quickly.